packages/protocol/contracts/tokenvault/BridgedERC20Base.sol

```diff
@@ -62,7 +64,7 @@ abstract contract BridgedERC20Base is EssentialContract, IBridgedERC20 {
         address _migratingAddress = migratingAddress;
         if (msg.sender == _migratingAddress) {
             // Inbound migration
-            emit MigratedTo(_migratingAddress, _account, _amount);
+            emit MigratedTo(msg.sender, _account, _amount, migratingInbound);
```

Comment:
I think the first address in MigratedTo is the token's address, not the user's address.

Reply:
This is the case where `msg.sender == _migratingAddress` so seems okay?

Though I think it's clearer to use `_migratingAddress` like it was. But the event name seems a bit confusing because `_migratingAddress` is actually the address that is migrated from (unless I misunderstand, calling the event `Migrated` now that it also contains the `migratingInbound` maybe more clear if I understand correctly at least.

Reply:
Lets change it to `Migrated`